### PR TITLE
Fixed minor spelling and grammatical errors, in addition to rephrasin…

### PR DESCRIPTION
**Pull request by MaxGreenspan**
Fixed minor spelling and grammatical errors.
Rephrased the Breadth First Traversal Practice section directions.

### CODE REVIEW

File: experiment/breadth-first-traversal/simulation/bft-exercise.html [hunk 1]
```diff
@@ -32,9 +30,9 @@
      </div>
 
 <p id="que">
-<b>Question:</b>Suppose a new kind of BFT traversal such that 
+<b>Question:</b> Suppose there is a new kind of BFT traversal such that 
 the traversal at the particular height is from right to left instead of left to right and this new BFT traversal is called Sudo BFT.
```
Comment (raj-vlabs) [inline experiment/breadth-first-traversal/simulation/bft-exercise.html:34]: I think they mean pseudo BFT and not Sudo BFT

File: experiment/breadth-first-traversal/simulation/bft-exercise.html [hunk 2]
```diff
@@ -11,15 +11,13 @@
...
-       <li>Click <b>Generate Graph</b> to generate a random
-       graph.
-       <li>Select a traversal.</li>
-       <li>Click on the nodes in the correct sequence of
-       selected traversal.</li>
-       <li>Click <b>Reset</b> after completing each
-       traversal, before moving on to next traversal.</li>
-       <li>Click on text inside the circle to select the
-       node for traversal</li>
+      <li>Perform a Sudo BFT (see question below).</li>
```
Comment (raj-vlabs) [inline experiment/breadth-first-traversal/simulation/bft-exercise.html:14]: Pseudo BFT in place of Sudo BFT?

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
Changes are generally fine, please take a look at a couple of comments

File: experiment/breadth-first-traversal/simulation/bft-exercise.html (hunk 2 above)
Comment (MaxGreenspan) [reply experiment/breadth-first-traversal/simulation/bft-exercise.html]: Yes, I think you're right.

File: experiment/breadth-first-traversal/simulation/bft-exercise.html [hunk 3]
```diff
@@ -11,15 +11,13 @@
...
-       <li>Select a traversal.</li>
-       <li>Click on the nodes in the correct sequence of
-       selected traversal.</li>
-       <li>Click <b>Reset</b> after completing each
-       traversal, before moving on to next traversal.</li>
-       <li>Click on text inside the circle to select the
-       node for traversal</li>
+      <li>Perform a pseudo BFT (see question below).</li>
+      <li>Click on the nodes in the correct sequence of
+        a Pseuudo BFT traversal.</li>
```
Comment (raj-vlabs) [inline experiment/breadth-first-traversal/simulation/bft-exercise.html:16]: Please fix the typo in the spelling of pseudo

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

**Pull request merged**